tests/ui/arch/atomics/atomic_flag_clear.rs

```diff
@@ -0,0 +1,25 @@
+// build-pass
+
+#![allow(incomplete_features)]
+#![feature(generic_const_exprs)]
+use spirv_std::{
+    arch::IndexUnchecked,
+    memory::{Scope, Semantics},
+};
+
+#[cfg(target_arch = "spirv")]
+use core::arch::asm;
+
+#[spirv(compute(threads(64)))]
+pub fn main(#[spirv(descriptor_set = 0, binding = 0, storage_buffer)] buffer: &mut [u32]) {
+    // Ensure kernel capabilities.
+    unsafe { asm!("OpCapability Kernel") };
```

Comment:
Do we have other tests that do this? My understanding was that we've given up on `Kernel` support (could be wrong though).

Reply:
`Kernel` capability is required for the atomic flag operations. Other test that require specific capabilities are the atomic floats, which I've omitted for the reason that the capability is vendor specific (Nvidia only if I'm correct). The same can be done for the atomic flags tests if required